src/pages/apply.js

```diff
@@ -21,34 +21,41 @@ export default function Apply() {
                 <div className="contactus-brief">
                   <div className="container-fluid">
                     <p className="section-description">
-                      Thank you for choosing to apply to Vets Who Code. Your first step in this
-                      journey will be to visit our&nbsp;
-                      <a href="https://github.com/Vets-Who-Code/prework">prework repository</a>
-                      &nbsp;on <a href="http://github.com">github.com</a>. We ask that, prior to
-                      applying to our program, you complete a small series of tutorial assignments
-                      that will introduce you to the basics of HTML5, CSS3 and JavaScript. The
-                      prework reading assignment will also guide you through setting up your
-                      development environment to work with the Vets Who Code program. After
-                      finishing the reading, we ask that you complete the capstone project&nbsp;(a
-                      short one-page website which will allow us to gauge your initial skill levels
-                      and help us assign an appropriate mentor), and fill out the following
-                      application form.
+                      Thank you for choosing to apply to Vets Who Code. This application is our
+                      flaship web development course. Here&apos;s a link to the{' '}
+                      <Link href="/syllabus">syllabus</Link>. To apply to the course fill out the
+                      Google form linked below. If you&apos;re already registered to the VetsWhoCode
+                      Slack channel we&apos;ll just ask you for your pre-work url and Slack name.
+                    </p>
+
+                    <p>
+                      If you have any questions about the pre-work, please reach out to us either in
+                      Slack or via the <Link href="/contact">Contact</Link> form.
+                    </p>
+
+                    <p className="section-description">
+                      <a
+                        target="_blank"
+                        rel="noreferrer"
+                        href="https://dev.to/vetswhocode/vets-who-code-pre-work-1gld"
+                      >
+                        Pre-work instructions
+                      </a>
                     </p>
                   </div>
                   <div className="container-fluid">
                     <div className="text-center">
                       <a
                         className="btn btn-charity-default"
-                        href="https://github.com/Vets-Who-Code/prework"
+                        href="https://forms.gle/YJ4c42QBaKo4fWXH9"
```

Comment:
This form isn't public yet.

![image](https://user-images.githubusercontent.com/38958206/216335465-f8efcd68-5e15-44f5-a4ec-2d5ec5921f34.png)

Reply:
Google Form permissions are strangely worded. I think I fixed it, can you try again?

Reply:
Works!